tests/progs/test_jubjub.py

```diff
@@ -191,3 +191,20 @@ def test_basic_point_functionality():
             await p1.montgomery_mul(5))
 
     await run_test_prog(_prog, test_preprocessing)
+
+
+@mark.asyncio
+async def test_share_mul(test_preprocessing):
```

Comment:
Try adding more preprocessing elements to get this test to pass-- 
`run_test_prog(_prog, test_preprocessing, k=5000)`

Reply:
(that's the context I've seen this error in at least)

Reply:
Will try, thanks!